docs/services/sensors.md

```diff
@@ -29,7 +26,7 @@ Go to your robot's **Code sample** tab on the [Viam app](https://app.viam.com) f
 
 {{% /alert %}}
 
-### Sensors
+### GetSensors
```

Comment:
```suggestion
### Sensors
```
I think this needs to stay `Sensors`, as least as long as it's defined so in the interface here https://pkg.go.dev/go.viam.com/rdk/services/sensors#Service and client file. Same for `Readings`!